sdk/python/feast/cli.py

```diff
@@ -423,5 +423,100 @@ def serve_command(ctx: click.Context, port: int):
     store.serve(port)
 
 
+@cli.group(name="alpha")
+def alpha_cmd():
+    """
+    Access alpha features
+    """
+    pass
+
+
+@alpha_cmd.command("list")
+def list_alpha_features():
```

Comment:
Nit, would be a good idea to show the status (enabled/disabled) of these flags in this command.

Reply:
Done